post-scan-actions/azure-python-teams-notification/teamsNotification/ScanResultHandler/handler.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import urllib3
+import json
+import os
+http = urllib3.PoolManager()
+import azure.functions as func
+
+# Hide verbose HTTP logging
+logging.getLogger('azure.core.pipeline.policies.http_logging_policy').setLevel(logging.WARNING)
+
+
+def main(message: func.ServiceBusMessage):
+    # set teams variables
+    url = os.environ['TEAMS_URL_WEBHOOK']
+    account = os.environ["TENANT_NAME"]
+
+    # Log the Service Bus Message as plaintext
+    message_body = message.get_body().decode("utf-8")
+    logging.info('Python ServiceBus topic trigger processed message.')
+    logging.info(f'Message Body: {message_body}')
+
+    message = json.loads(message_body)
+    findings = message['scanning_result'].get('Findings')
+
+    if findings:         
+        malwares = []
+        types = []
+        for finding in findings:
+            malwares.append(finding.get('malware'))
+            types.append(finding.get('type'))
+            
+        malwares=', '.join(malwares)
+            
+        types=', '.join(types)
+            
+        file_url=str(message['file_url'])
+            
+            
+        payload = {
```

Comment:
Please adjust the indents of the whole dictionary to align with the python file.

Reply:
Sorry battling VS code auto formatting it. I adjusted the indents.